janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -3518,6 +3518,10 @@ public void testTinkerPopOptimizationStrategies() {
         TraversalMetrics metrics;
         GraphTraversalSource gts = graph.traversal();
 
+        clopen(option(USE_MULTIQUERY), true);
+        gts = graph.traversal();
+        List<Vertex> results = gts.V(sv[0]).emit().repeat(__.barrier().out("knows")).toList();
```

Comment:
verify the results

Reply:
This was leftover from some debugging and shouldn't have been included. I'll get rid of it as this combo of steps is covered by the TP test suite.